docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,162 @@
+
+```python file=/guides/dagster/automating_pipelines/vanilla_schedule.py
+from dagster import AssetSelection, define_asset_job, ScheduleDefinition
+
+asset_job = define_asset_job("asset_job", AssetSelection.groups("some_asset_group"))
+
+basic_schedule = ScheduleDefinition(job=asset_job, cron_schedule="0 9 * * *")
+```
+
+You can also use the <PyObject object="schedule" decorator /> decorator if you want to provide custom run config and tags. For example, if you want to pass a parameter to the job at runtime, for example, activity = ‘party’ on the weekend and activity=’grind’ on the weekdays, you can use the configuration to pass parameters in.
+
+```python file=/guides/dagster/automating_pipelines/config_schedule.py
+from dagster import schedule, ScheduleEvaluationContext, RunRequest
+
+##sets the schedule to be updated everyday at 9:00 AM
+
+@schedule(job=configurable_job, cron_schedule="0 9 * * *")
+def configurable_job_schedule(context: ScheduleEvaluationContext):
+  if context.scheduled_execution_time.weekday()<5:
+    activity_selection='grind'
+  else:
+    activity_selection= 'party'
+  return RunRequest(
+    run_config={"ops": {"configurable_op": {"config": {"activity": activity_selection}}}})
+```
+
+### I want my data to be updated at different cadences
+
+Let’s say the CFO wants a report updated by 9:00 AM so they can drink their coffee and review the latest numbers. The data comes from a number of systems and are all part of your DAG.
+
+- Some of the data, such as last quarter’s earnings report to investors, only need to be updated every 3 months.
+- Other data, such as the most recent revenue, needs to be updated every day and the CFO wants to see how the company did yesterday.
+
+Dagster’s [declarative scheduling](/guides/dagster/scheduling-assets) allows you to set a freshness policy. The decision to use declarative scheduling and freshness policies is not a purely practical decision but also a mindset change compared to imperative workflow processes.
+
+For example, using a 30 minute freshness policy, the asset would incorporate all data that arrived 30 minutes before that time.
+
+This approach allows you to think about each asset’s freshness requirements individually. Then, Dagster figures out how to get the report updated by 9:00AM every day for the CFO.
+
+In the following example, we set the freshness policy to instruct the sales asset not to be more than 60 minutes old by 9:00 AM. The sales data will prompt the upstream transactions asset to materialize to stay within the freshness policy.
+
+We also have an asset for the expenses, which also depends on the transactions asset. This asset has a freshness policy of 120 minutes, or two hours at any time, since it did not specify a schedule. The expenses asset will require the transactions data to be refreshed in order to comply with the freshness policy.
```

Comment:
```suggestion
We also have an asset for the expenses, which also depends on the transactions asset. This asset has a freshness policy of 120 minutes, or two hours at any time, since it did not specify a schedule. The `expenses` asset will require the `transactions` data to be refreshed in order to comply with the freshness policy.
```

🔧 Formatting the names of the assets as code - it makes them stand out and also implies they're present in the code that follows